recipe/build.sh

```diff
@@ -30,6 +30,7 @@ RELEASE_DIR=$PYPY3_SRC_DIR/pypy/tool/release
 PYPY_PKG_NAME=pypy3
 BUILD_DIR=${PREFIX}/../build
 TARGET_DIR=${PREFIX}/../target
+export MACOSX_DEPLOYMENT_TARGET=10.13
```

Comment:
That's not going to help. It needs to be set in `conda_build_config.yaml` and then rerendered